servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -207,26 +243,22 @@ public void stop() throws IOException {
   }
 
   /**
-   * Completes a file checkpoint in ufs. Called via RPC.
+   * Persists a file in UFS.
    *
-   * @param workerId the worker id completing the ufs checkpoint
-   * @param fileId the file id associated with the ufs checkpoint
+   * @param fileId the file id
    * @param length the length of the file
-   * @param checkpointPath the ufs path of the file
    * @return true on success
    * @throws SuspectedFileSizeException
    * @throws BlockInfoException
    * @throws FileDoesNotExistException
    */
-  public boolean completeFileCheckpoint(long workerId, long fileId, long length,
-      TachyonURI checkpointPath)
-          throws SuspectedFileSizeException, BlockInfoException, FileDoesNotExistException {
+  public boolean persistFile(long fileId, long length) throws SuspectedFileSizeException,
+      BlockInfoException, FileDoesNotExistException {
     synchronized (mInodeTree) {
       long opTimeMs = System.currentTimeMillis();
-      LOG.info(FormatUtils.parametersToString(workerId, fileId, length, checkpointPath));
-      if (completeFileCheckpointInternal(workerId, fileId, length, checkpointPath, opTimeMs)) {
+      if (persistFileInternal(fileId, length, opTimeMs)) {
```

Comment:
I see, I believe that if ``persistFileInternal(fileId, ...)`` returns true, then it is true that ``inode.isPersisted`` is true for an ``inode`` that matches the ``fileId``. It could be that we log extraneous PersistFile events, but they won't affect the correctness of the state. Does that make sense?